<launch>
    <group>
        <node pkg="joy" type="joy_node" name="joy_node">
            <param name="dev" type="string" value="/dev/input/js0" />
            <param name="deadzone" type="double" value="0.3" />
            <remap from="joy" to="tiago_dual_cartesio/joy" />
            <param name="autorepeat_rate" type="double" value="20" />
        </node>
    </group>

    <node pkg="stream_deck_controller" type="joystick_controller.py" name="joystick_controller" output="screen">
        <param name="joy_topic" type="string" value="/streamdeck/joy" />
        <param name="lin_scale" type="double" value="0.2" />
        <param name="ang_scale" type="double" value="0.5" />
    </node>
</launch>